Userland/Libraries/LibWeb/DOM/ParentNode.h

```diff
@@ -28,6 +28,9 @@ class ParentNode : public Node {
 
     JS::NonnullGCPtr<HTMLCollection> children();
 
+    JS::NonnullGCPtr<HTMLCollection> get_elements_by_tag_name(FlyString const&);
+    JS::NonnullGCPtr<HTMLCollection> get_elements_by_tag_name_ns(Optional<String> const&, FlyString const&);
```

Comment:
Previosuly was a `DeprecatedString`, but since a `Optional<String>` is passed through we can't make this an `Optional<FlyString>`. Wondering if it's worth adding an IDL abbtritute to generate this as one, or make it implicitly convertable :thonk:

Reply:
Hmm, feels like something we don't *have* to figure out right now.
It would be nice if it can take `Optional<FlyString>` ofc, but that will indeed require more smarts in the IDL generator stage.